packages/babel-types/src/index.js

```diff
@@ -240,6 +240,16 @@ export function clone(node: Object): Object {
 }
 
 /**
+ * Create a shallow clone of a `node` excluding `_private` and location properties.
```

Comment:
`clone` excludes the private properties. Would adding an options field to clone be a better option vs. a separate named function? I suspect that there are other places where this without location method will be needed, I just don't have time to identify all of the cases.